### ISSUE TITLE
Allow response mocking for unit tests

### ISSUE THREAD

**Issue by phortx** [vuex-orm/plugin-graphql#57]
This plugin should provide some tools to allow the simple unit testing of components which interact with the GraphQL API.

**Comment by phortx** [comment]
I'm thinking of something like

```js
GraphQLMockTools.mock('fetch', { id: 15 }).for(User).andReturn({ name: 'Charlie Brown' });
GraphQLMockTools.mock('mutate', { mutation: 'upvote', variables: { id: 42 } }).for(Post).andReturn({ name: 'Example Post', upvotes: 4 });
```

to mock a `User.fetch(15)`.

**Comment by phortx** [comment]
Released with v1.0.0-RC.17, see https://vuex-orm.github.io/plugin-graphql/guide/testing/
